src/i18n/translations/en/translation.json

```
@@ -1 +1,14 @@
-{}
\ No newline at end of file
+{
+  "404": {
+    "h1": "404",
+    "text": "<0>The URL you entered may be broken, or the page has been removed.</0><1> </1><2>Go back to the home page</2>"
```

I _think_ this can be simplified to:
```suggestion
    "text": "The URL you entered may be broken, or the page has been removed.<2>Go back to the home page</2>"
```

---

Same with the two things in the `dataDictionary` section: if the element is just a plain text, it doesn't look like we need to wrap it.

I realize that this might have been auto-generated tho, so maybe it's not worth worrying about.

---

Thanks! I started using the extraction tool after doing this one manually. Made the change